packages/web3/test/fixtures/config.ts

```diff
@@ -0,0 +1,41 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+type Account = {
+	address: string;
+	privateKey: string;
+	balance: string;
+};
+
+export const accounts: Account[] = [
+	{
+		address: '0xdc6bad79dab7ea733098f66f6c6f9dd008da3258',
+		privateKey: '0x4c3758228f536f7a210f8936182fb5b728046970b8e3215d0b5cb4c4faae8a4e',
+		balance: '100',
+	},
+	{
+		address: '0x962f9a9c2a6c092474d24def35eccb3d9363265e',
+		privateKey: '0x34aeb1f338c17e6b440c189655c89fcef148893a24a7f15c0cb666d9cf5eacb3',
+		balance: '100',
+	},
+];
+export const httpStringProvider = 'http://localhost:8545';
+export const wsStringProvider = 'ws://localhost';
+export const ipcStringProvider = '../../../../../.github/geth/data/geth.ipc';
+export const externalHttpsStringProvider =
+	'https://mainnet.infura.io/v3/6120b2e72c304b4eadafe2bf33862ac4';
```

Comment:
infura keys shouldn't be exposed

Reply:
So if it is committed once - it is saved in git forever. So now you have only one way. Remove and create a new one.

Reply:
If it's in your branch, you can rewrite the commits and force-push to change the history.

Reply:
Yes it is a node of yours, but your node is exposed on github on a huge project and may lead to vulnerabilities if someone decides to use your node. For example if a bad actor maliciously uses your node to continuosly spam requests, they may hit your infura limit and our testcases using your node will fail.
In this case I think it is best to use an [environment variable](https://docs.github.com/en/actions/learn-github-actions/environment-variables) to store infura nodes, I believe this is what is currently being done with infura keys in 1.x